Game/Src/TestEnemyMovement.gd

```diff
@@ -229,38 +195,11 @@ func spawn_player():
 	tween.interpolate_property(newPlayer, "position:x",PLAYER_SPAWN_X,242,2,Tween.TRANS_CUBIC, Tween.EASE_IN)
 	tween.start()
 
-func _on_timer_timeout():
-	var idx = rng.randi_range(0, 3)
-	emit_signal("glory_increased",-gloryLoss)
-	if toggle1.isEmpty:
-		toggle1.texture_id = idx
-		power_slots[0] = idx#powers.keys()[idx]
-		toggle1.change_to_filled_texture()
-		return
-	if toggle2.isEmpty:
-		toggle2.texture_id = idx
-		power_slots[1] = idx#powers.keys()[idx]
-		toggle2.change_to_filled_texture()
-		return
-
-func lightning():
-	if closest_enemy != null:
-		closest_enemy.die()
-
-func slippery_hands():
-	emit_signal("glory_increased",-10)
+func _on_Power1Toggle_done_filling():
```

Comment:
If we use the enum 0 = empty, this becomes useless.

Reply:
Not sure, a signal is sent whenever a toggle is done filling. Then, we check whether a toggle is empty in order to start a new cooldown. This is basically my way to replace the timer that was in testenemymovement (I'm really looking forward to trimming this scene as much as possible) and ensuring that one toggle is filling at the time